distribution/docs/src/main/resources/_contents/_endpoints/metrics-endpoint-contents.adoc

```diff
@@ -0,0 +1,221 @@
+==== Metrics Endpoint
+
+[NOTE]
+====
+EXPERIMENTAL
+====
+
+[WARNING]
+====
+Note that the Metrics endpoint URL says "internal."
+This indicates that this endpoint is intended for internal use by the ${branding} code.
+This endpoint is likely to change in future versions; therefore, any custom applications built to make use of it, as described below, should be made with caution.
+====
+
+The Metrics Endpoint is used by the <<Metrics Collection Application>> to report on system metrics.
+
+===== Installing the Metrics Endpoint
+
+The Metrics Endpoint is installed by default with a standard installation in the ${ddf-platform} application.
+
+===== Configuring the Metrics Endpoint
+
+No configuration can be made for the Metrics Endpoint.
+All of the metrics that it collects data on are pre-configured in ${branding}.
+
+===== Using the Metrics Endpoint
+
+.Metrics Endpoint URL
+[source,http]
+----
+${secure_url}/services/internal/metrics/catalogQueries.png?startDate=2013-03-31T06:00:00-07:00&endDate=2013-04-01T11:00:00-07:00
+----
+
+The table below lists all of the options for the Metrics endpoint URL to execute custom metrics data requests:
+
+.Metrics Endpoint URL Options
+[cols="1m,5,3" options="header"]
+|===
+
+|Parameter
+|Description
```

Comment:
To be consistent with other tables, should we have a `Required` column and remove the _(optional)_ from the last two parameter descriptions?